android/app/src/main/AndroidManifest.xml

```diff
@@ -22,5 +22,8 @@
             <category android:name="android.intent.category.LAUNCHER" />
         </intent-filter>
       </activity>
+      <meta-data  
+        android:name="com.google.android.geo.API_KEY"  
+        android:value="AIzaSyA5r1HhX9E0UtWvsslqfmuQj-fFFaaUgQA"/>
```

Comment:
⚠ As a heads up storing keys in the codebase could be dangerous. ⚠
There are many script bots that scrape through public repos and steal API keys. This shouldn't really be a problem for us with Google Maps API key as there isn't much damage that can be done with it and the owner (Brenden can also secure the key on his side ensuring only this app can use the key. However, you should always be wary of adding keys directly into the code. Instead, create another file and place that file in a `.gitIgnore` or find another way to secure it.

Reply:
Noted, Touching base with Brenden on this and will create another PR to address this if necessary.